uint8 turn_signal_pos
uint64 timestamp 
uint64 source_time
